device/linkstate/teprofile.yaml

```diff
@@ -0,0 +1,76 @@
+components:
+  schemas:
+    LinkState.TE:
+      description: >-
+        A container for Traffic Engineering properties on a interface.
+      type: object
+      properties:
+        administrative_group:
+          description: |-
+            The Administrative group sub-TLV (sub-TLV 3). It is a 4-octet 
+            user-defined bit mask used to assign administrative group numbers 
+            to the interface, for use in assigning colors and resource classes. 
+            Each set bit corresponds to a single administrative group for this 
+            interface. The settings translate into Group numbers, which range 
+            from 0 to 31 (integers).
+          type: string
+          format: hex
+          default: '00000000'
+        metric_level:
+          description: |-
+            The user-assigned link metric for Traffic Engineering.
+          type: integer
+          default: 0
+        max_bandwith:
+          description: |-
+            The maximum link bandwidth (sub-TLV 9) in bytes/sec allowed for this 
+            link in this direction.
+          type: integer
```

Comment:
Min/Max not specified.

Reply:
of its integer type an support upto maxInt then setting min value should be ok?

Reply:
Might be, assuming that the state-machine accepts those values.  Does the state-machine impose any additional restrictions?

Reply:
should not.